### DISCUSSION TITLE
how to only set the boundary condition for part of the axis.

### DISCUSSION THREAD

**Discussion by judy2016hh** [zwicker-group/py-pde#402] [Q&A]
Thanks for your great code! 
I met a little problem and didn't find the solution in the document. I try to solve a 2D Laplace problem, where the boundary condition is inhomogeneous and is given by an array when x>1 and no constraint when x<1. 

Is there any way to implement this kind of boundary condition?

**Comment by david-zwicker** [comment]
Please specify more details of your problem together with example code. As written right now, I have no idea what you're trying to achieve.

**Reply by judy2016hh** [reply]
Thank for your response. It is a 2D Laplace problem for $u(x,y)$, $x,y \in [0,2]$. The boundary condition is $u(x,y=0)=1$ when $x \in [1,2]$.

**Reply by david-zwicker** [reply]
Please provide example code of what you have tried.
